string new_human_state
---
bool success
